Framework/CurveFitting/src/FuncMinimizers/DampedGaussNewtonMinimizer.cpp

```diff
@@ -30,6 +30,7 @@ DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped GaussNewton)
 DampedGaussNewtonMinimizer::DampedGaussNewtonMinimizer(double relTol)
     : IFuncMinimizer(), m_relTol(relTol) {
   declareProperty("Damping", 0.0, "The damping parameter.");
+  declareProperty("Debug", false, "Turn on the debug output.");
```

Comment:
In any case I think it is a question for a separate issue. Please feel free to open one. Here if you don't mind I'll rename the property and keep the rest as it is.